prow/cmd/hook/main.go

```diff
@@ -46,6 +46,7 @@ var (
 
 	presubmit    = flag.String("presubmits", "/etc/jobs/presubmit", "Path to presubmit.yaml.")
 	postsubmit   = flag.String("postsubmits", "/etc/jobs/postsubmit", "Path to postsubmit.yaml.")
+	cron         = flag.String("crons", "/etc/jobs/cron", "Path to cron.yaml.")
```

Comment:
I suppose, but it's not the same format at all. It might be more confusing to call it that...